src/UI/Factory.php

```diff
@@ -261,15 +261,15 @@ public function image();
 	 *     This component is used to wrap an existing ILIAS UI element into a UI component. This is useful if a container
 	 *     of the UI components needs to contain content that is not yet implement in the centralized UI components.
 	 *   composition: >
-	 *     The generic component contains html or any other content as string.
+	 *     The legacy component contains html or any other content as string.
 	 *
 	 * rules:
 	 *   wording:
 	 *      1: This component MUST only be used to ensure backwards compatibility with existing UI elements in ILIAS.
```

Comment:
Thank you for your feedback. I personally appreciate your suggestion to stress this point further. 

Rule changes MUST be discussed by the JF. In this case however, I do not believe this to be an actual change of the rules, but a clarification of a rule already listed. I therefore commit your suggestion.